_data/simple-icons.json

```
@@ -3563,6 +3563,12 @@
             "hex": "4A86CF",
             "source": "https://flathub.org/"
         },
+        {
+            "title": "Flatpak",
+            "hex": "4A90D9",
+            "source": "https://flatpak.org/",
+            "guidelines": "https://flatpak.org/img/flatpak-logos.zip"
```

According to the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), the link to the branding page and not the link to the ZIP archive should be used:

> If the SVG is sourced from:
>
> - **Branding page**: For an SVG from a branding page the source URL should link to the branding page and not the image, PDF, or archive (such as `.zip`) file.

```suggestion
            "guidelines": "https://flatpak.org/press/"
```